### PR TITLE
Comply with wildcard regulations

### PR DESCRIPTION
**Pull request by JamesHawkinss**
# Pull request

 - [x] I have read and agreed to the [CODE OF CONDUCT](https://github.com/CascadeBot/CascadeBot/blob/master/CODE_OF_CONDUCT.md).
 - [ ] What I'm implementing was assigned to me or was not being worked on and is a somewhat requested feature. For reference see our [trello](https://trello.com/b/gUQehy4l/bot).
 - [x] I have tested all of my changes.
 - [x] I have followed the style of the rest of the project.
 
## Added/Changed
What did you add and/or change for this.

## Feature
If this is a new feature, why should we add it to the bot.

### CODE REVIEW

File: src/main/java/com/cascadebot/cascadebot/commands/fun/CatCommand.java [hunk 1]
```diff
@@ -9,31 +9,30 @@
...
 import org.json.JSONArray;
 import org.json.JSONObject;
 import java.io.IOException;
 
-
 public class CatCommand implements ICommandMain {
 
     @Override
     public void onCommand(Member sender, CommandContext context) {
-        OkHttpClient client = new OkHttpClient();
```
Comment (binaryoverload) [inline src/main/java/com/cascadebot/cascadebot/commands/fun/CatCommand.java]: No blank line here :D

Comment (JamesHawkinss) [reply src/main/java/com/cascadebot/cascadebot/commands/fun/CatCommand.java]: There is on my screen? and no files to commit so eh?

Comment (binaryoverload) [reply src/main/java/com/cascadebot/cascadebot/commands/fun/CatCommand.java]: I mean there shouldn't be a blank line here :P

**Comment by JamesHawkinss** [comment]
> While you're here, can you create actual permissions for these? They should be able to be restricted

all done xoxo

**Pull request merged**